README.md

```diff
@@ -84,8 +85,22 @@ After running the commands given in the [Building](#building) section of this do
 the configuration can be updated as follows:
 ~~~~
 cd build
-cmake .. -DCONFIG_CALIBRATE_PLL=<true|false> -DCONFIG_CALIBRATE_ZQCAL=<true|false> -DCONFIG_CALIBRATE_SA=<true|false>
+cmake .. -DCONFIG_CALIBRATE_PLL=<true|false> -DCONFIG_CALIBRATE_ZQCAL=<true|false> -DCONFIG_CALIBRATE_SA=<true|false> -DCONFIG_DRAM_TRAIN=<true|false>
```

Comment:
We need to add support for this option in the configure script. I think by default, it should be off. The flag will enable training.

Reply:
Added this to the configure script.  Updated in the CMakeLists.txt to be false by default and did the same in the configure script.  Specifying --dram-train with the configure script will enable training by setting that to true.